modules/wechat_qrcode/src/wechat_qrcode.cpp

```diff
@@ -119,9 +119,10 @@ vector<string> WeChatQRCode::Impl::decode(const Mat& img, vector<Mat>& candidate
     vector<string> decode_results;
     for (auto& point : candidate_points) {
         Mat cropped_img;
+		Align *aligner = NULL;
         if (use_nn_detector_) {
-            Align aligner;
-            cropped_img = cropObj(img, point, aligner);
+            aligner = new Align();
```

Comment:
Please avoid using operator new(). You should use `cv::Ptr` or `std::shared_ptr`:
```
auto aligner = makePtr<Align>();
```